pyproject.toml

```diff
@@ -54,7 +54,8 @@ processing = [
     "tldextract",
     "trafilatura>=1.8.0",
     "tokenizers",
-    "ftfy"
+    "ftfy",
+    "spacy",
```

Comment:
SpaCy dependency for the Korean language tokenizer is currently in the "processing" group. However, as more tokenizers are added in the future, we would like to minimize the number of dependencies that the user has to download if they do not need to process all languages.